pjmedia/include/pjmedia/rtcp.h

```diff
@@ -186,6 +186,8 @@ typedef struct pjmedia_rtcp_stream_stat
     unsigned        loss;       /**< Total number of packets lost           */
     unsigned        reorder;    /**< Total number of out of order packets   */
     unsigned        dup;        /**< Total number of duplicates packets     */
+    unsigned        nack_cnt;   /**< Total number of NACK packets           */
```

Comment:
May be `Total number of requested packets using NACK`?

Reply:
I updated the comment